### PR TITLE
Add amdgpu-pro-opencl-20.10.1048554.ebuild

### PR DESCRIPTION
**Pull request by madgagarin**
Add new version amdgpu-pro-opencl

Download package from AMD site manually.
https://www.amd.com/en/support/kb/release-notes/rn-amdgpu-unified-linux-20-10

### CODE REVIEW

File: dev-libs/amdgpu-pro-opencl/amdgpu-pro-opencl-20.10.1048554.ebuild [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+LICENSE="AMD-GPU-PRO-EULA"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RESTRICT="bindist mirror fetch strip"
+
+BDEPEND="dev-util/patchelf"
+COMMON=">=virtual/opencl-3"
+DEPEND="${COMMON}"
+RDEPEND="${COMMON}
```
Comment (juippis) [inline dev-libs/amdgpu-pro-opencl/amdgpu-pro-opencl-20.10.1048554.ebuild:26]: `COMMON` what? Rename to `COMMON_DEPEND` that's widely used.

Comment (madgagarin) [reply dev-libs/amdgpu-pro-opencl/amdgpu-pro-opencl-20.10.1048554.ebuild:26]: Ok

**Comment by rossbridger** [comment]
The two commits should be consolidated into one and be signed off using repoman

**Comment by ajakk** [comment]
Obsolete since a while

**Pull request closed without merging**